Sources/Rendering/Core/Camera/index.js

```diff
@@ -426,6 +426,38 @@ export function extend(publicAPI, model, initialValues = {}) {
 
   // Build VTK API
   macro.obj(publicAPI, model);
+
+  if (model.position === null) {
+    model.position = [0, 0, 1];
+  }
+  if (model.focalPoint === null) {
+    model.focalPoint = [0, 0, 0];
+  }
+  if (model.viewUp === null) {
+    model.viewUp = [0, 1, 0];
+  }
+  if (model.directionOfProjection === null) {
+    model.directionOfProjection = [0, 0, -1];
+  }
+  if (model.clippingRange === null) {
+    model.clippingRange = [0.01, 1000.01];
+  }
+  if (model.windowCenter) {
+    model.windowCenter = [0, 0];
+  }
+  if (model.viewPlaneNormal === null) {
+    model.viewPlaneNormal = [0, 0, 1];
+  }
+  if (model.screenBottomLeft === null) {
+    model.screenBottomLeft = [-0.5, -0.5, -0.5];
+  }
+  if (model.screenBottomRight === null) {
+    model.screenBottomRight = [0.5, -0.5, -0.5];
+  }
+  if (model.screenTopRight === null) {
```

Comment:
By initializing it to null a check against null means it has not been set. I'm not sure what leaving it uninitialized and then testing via ! does.  Put another way, I'm pretty sure the way I am doing it works and I am not sure other ways would work. Maybe they would. I just am not that smart.